### PR TITLE
Complete 64bit support

### PR DESCRIPTION
**Pull request by CMTaylor**
This pull request completes the 64-bit build support work done by squidpunch and submitted as pull request #37 on TestingWithFrank/Frank. It includes setting the 64-bit option for Debug builds and setting the 64-bit option in the PublicAutomation sub-module (that change submitted in a separate pull request on that module).

### CODE REVIEW

File: .gitmodules [hunk 1]
```diff
@@ -12,7 +12,7 @@
 	url = https://github.com/robbiehanson/CocoaHTTPServer.git
 [submodule "lib/PublicAutomation"]
 	path = lib/PublicAutomation
-	url = https://github.com/TestingWithFrank/PublicAutomation.git
```
Comment (moredip) [inline .gitmodules]: I've pulled in your PublicAutomation fix. Please restore this submodule to point back to the `TestingWithFrank/PublicAutomation` repo

File: gem/frank-cucumber.gemspec [hunk 2]
```diff
@@ -22,13 +22,13 @@ Gem::Specification.new do |s|
 
   s.add_dependency( "cucumber", ["~>1.3.10"] )
   s.add_dependency( "rspec", ["~>2.14.1"] )
-  s.add_dependency( "sim_launcher", ["~>0.4.6"] )
```
Comment (moredip) [inline gem/frank-cucumber.gemspec]: were these dependency updates necessary as part of the 64 bit work? If not I'd like them in a separate Pull Request.

**Comment by moredip** [comment]
Thanks for the contribution @CMTaylor. 

I'd love to pull in the 64 bit support but as this Pull Request stands today it also includes other changes (status bar orientation, direct app exec, dependency version updates, etc). Can you send me a Pull Request that just has the changes for 64 bit support?

**Comment by squidpunch** [comment]
@moredip I think my original PR gave just the 64-bit support, was there anything missing there? #37

**Comment by CMTaylor** [comment]
I have great difficulty with GIT, so maybe taking David’s and fixing it, as I tried to do, would be best. Hi PR was missing the dual architecture for Debug builds and missing it for PublicAutomation, which I think you’ve already taken from my PR on that repository.

Cheers,
Martin

**Comment by moredip** [comment]
Closing this, PR #42 handled the 64 bit stuff.

**Pull request closed without merging**